pkgs/applications/networking/browsers/luakit/default.nix

```diff
@@ -0,0 +1,85 @@
+{ stdenv
+, fetchurl
```

Comment:
Put these on multiple lines, but not one word per line, please. Group them a little.